samples/cpp/speech_sample/main.cpp

```diff
@@ -89,19 +92,26 @@ int main(int argc, char* argv[]) {
         std::vector<std::string> output_names;
         std::vector<size_t> ports;
         // --------------------------- Processing custom outputs ---------------------------------------------
-        if (!FLAGS_oname.empty()) {
-            output_names = convert_str_to_vector(FLAGS_oname);
-            for (const auto& output_name : output_names) {
-                auto pos_layer = output_name.rfind(":");
-                if (pos_layer == std::string::npos) {
-                    throw std::logic_error("Output " + output_name + " doesn't have a port");
-                }
-                outputs.push_back(output_name.substr(0, pos_layer));
-                try {
-                    ports.push_back(std::stoi(output_name.substr(pos_layer + 1)));
-                } catch (const std::exception&) {
-                    throw std::logic_error("Ports should have integer type");
-                }
+        std::pair<std::string, std::vector<std::string>> output_data;
+        std::pair<std::string, std::vector<std::string>> reference_data;
```

Comment:
The same question

Reply:
Because in this function we split the data into a list of names and a list of files passed in sequential order. So we don't need structure like as key:value